### PR TITLE
Loki-84: Fixed UI from User comments

### PR DESCRIPTION
**Pull request by rfxu**
-  On the Admin Queue page, in the Search box, the names of the searchable fields should be described
-  On the Admin Detail page, when no image has been uploaded, there should be a generic image to indicate
-  On the Photo Upload page, the text describing Step 1 should be "Enter the Unique Code that you received in an email. The Unique Code is 6 digits (example: 111111).
-  On each Admin page, there should be a link to Swagger and Git in the header/footer
-  On each page, the updated Loki Tracker logo should be displayed
-  On the Admin Detail page, there should be no scroll bars.

### CODE REVIEW

File: frontend/src/app/tpl-footer/tpl-footer.component.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
 
-<div class="row">
-	<div class="col-lg-6 color-gray"> © 2017 NCI Inc. All Rights Reserved.</div>
-	<div class="col-lg-6"></div>
+<div class="row color-gray">
+	<div class="col-lg-6"> © 2018 NCI Inc. All Rights Reserved.</div>
+	<div class="col-lg-6 text-right icons">
+		<b>Resources:</b> 
+		<a href="https://github.com/nciinc/loki-tracker" target="new"><img src="./assets/images/GitHub-Mark-64px.png" alt="Github logo" title="Go to LokiTracker Github"/></a>
+		<a href="/api" target="new"><img src="./assets/images/Swagger-logo.png" alt="Swagger Logo" title="Go to LokiTracker API"/></a>
```
Comment (barrelcrash) [inline frontend/src/app/tpl-footer/tpl-footer.component.html]: Might want to add text "Github" and "API" to make things more obvious.

File: frontend/src/app/tpl-header/tpl-header.component.html [hunk 2]
```diff
@@ -1,5 +1,11 @@
 <h1 class="sr-only">Header</h1>
 <div class="header">
-	<div class="hdr-left"><h2 class="sitename"><img class="site-logo" src="./assets/images/logo.gif" title="Magnify Glass Loki Tracker logo"/>Loki<b>Tracker</b></h2></div>
-	<div class="hdr-right"><a href="https://www.nciinc.com/" target="new"><img class="nci-logo" src="./assets/images/NCI_Logo_MAIN_45_width.png" title="NCI logo"/></a></div>
+	<div class="hdr-left">
+		<h2 class="sitename">
+			<a href="/" title="Goto Customer Upload page">
+				<img class="site-logo" src="./assets/images/logo.gif" alt="Magnify Glass Loki Tracker logo"/><img class="site-name" src="./assets/images/Loki-Tracker-Logo.svg"  height="60" alt="Loki Tracker"/>
```
Comment (barrelcrash) [inline frontend/src/app/tpl-header/tpl-header.component.html]: This is okay but only if we also have an "admin login" link on the photo upload page.

Comment (rfxu) [reply frontend/src/app/tpl-header/tpl-header.component.html]: We properly will not have it.  I can remove it.

**Comment by barrelcrash** [comment]
LGTM

**Pull request merged**